content/docs/04-clusters/01-public-cloud/06-tke.md

```diff
@@ -256,7 +269,7 @@ Palette encourages its uses to go with the Public Cluster endpoint access as of
 
 8. Review settings and deploy the cluster. Provisioning status with details of ongoing provisioning tasks is available to track progress.
 
-# Deleting a Tencent Cluster
+# Delete a Tencent Cluster
```

Comment:
@resmaa take a peek at how I created Delete a Cluster for the AWS and Azure section [[Preview](https://deploy-preview-1058--docs-spectrocloud.netlify.app/clusters/public-cloud/aws/create-cluster#deletinganawsiaascluster)] [[PR](https://github.com/spectrocloud/librarium/pull/1058)]

Reply:
nice